cbindings/blitzar_api.h

```diff
@@ -642,6 +642,33 @@ void sxt_fixed_packed_multiexponentiation(void* res, const struct sxt_multiexp_h
                                           const unsigned* output_bit_table, unsigned num_outputs,
                                           unsigned n, const uint8_t* scalars);
 
+/**
+ * Compute a varying lengthing multiexponentiation of scalars in packed format using a handle to
+ * pre-specified generators.
+ *
+ * On completion `res` contains an array of size `num_outputs` for the multiexponentiation
+ * of the given `scalars` array.
+ *
+ * An entry output_bit_table[output_index] specifies the number of scalar bits used for
+ * output_index and output_lengths[output_index] specifies the length used for output_index.
```

Comment:
Does `length` refer to the number of pre-specified generators are used during multiexponentiation for the specific `output_index`?

Reply:
Yes.